src/axom/mir/doxygen_mainpage.md

```diff
@@ -0,0 +1,46 @@
+MIR {#mirtop}
+=============
+
+Axom's [Mir](@ref axom::mir), (M)aterial (I)interface (R)econstruction component
+provides classes that can process Blueprint meshes with mixed materials into new
+Blueprint meshes such that there is a single material per zone. In addition, the
+MIR component provides useful building blocks for developing algorithms using
+Blueprint meshes. There are views that simplify dealing with Conduit data and
+utility algorithms for processing and constructing meshes.
+
+# Design goals {#goals}
```

Comment:
I don't remember the scope of `#goals`, but conservatively, it may be the entire Axom Sphinx doc set.  Please change "#goals", "#views", and "#utilities" to something like "#mirgoals", "#mirviews", and "#mirutilities".

Reply:
Fixed.